packages/era/.gitignore

```diff
@@ -0,0 +1 @@
+*.era
```

Comment:
Should we also exclude e2s files? Not sure but thought its worth bringing up the question
```suggestion
*.e2s
*.era
```